zeppelin-zengine/src/main/java/org/apache/zeppelin/util/Util.java

```diff
@@ -29,6 +30,9 @@
   private static final String PROJECT_PROPERTIES_VERSION_KEY = "version";
   private static final String GIT_PROPERTIES_COMMIT_ID_KEY = "git.commit.id.abbrev";
   private static final String GIT_PROPERTIES_COMMIT_TS_KEY = "git.commit.time";
+  private static final String ZEPPELIN_NOTE_FILE_EXTENSION = "zpln";
+  private static final int MAX_ZEPPELIN_NOTE_FILE_LENGTH = 127;
```

Comment:
I see. this 127 does it generally include extension though? I ask since the calculate below is adding extension length on top of this 127

my guess is we could probably limit to lower than 127 (say 110 or 100) - most people probably don't care.